client/app/visualizations/pivot/index.js

```diff
@@ -21,7 +22,19 @@ function pivotTableRenderer() {
         if ($scope.queryResult.getData() !== null) {
           // We need to give the pivot table its own copy of the data, because it changes
           // it which interferes with other visualizations.
-          data = angular.copy($scope.queryResult.getRawData());
+          data = [];
+          const columns = $scope.queryResult.getColumns();
+          const rows = $scope.queryResult.getData();
+          rows.forEach((row) => {
+            const rowObj = {};
+            columns.forEach((col) => {
+              const colName = getColumnCleanName(col.name);
+              const value = formatValue($filter, clientConfig, row[col.name], col.type);
```

Comment:
Two things that come to mind here:

1. Are we sure that converting numbers into strings will behave properly with PivotTable?
2. For dates, shouldn't we use some standard like (YYYY-MM-DD) regardless of configuration to solve #216?